Styles/styles.css

```diff
@@ -0,0 +1,50 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.flex-container {
+  display: flex;
+  background-color: #10367c;
+  width: 1440px;
+  height: 630px;
+  color: #ffffff;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+.item {
+  width: 33.3333%;
+}
+li {
+  list-style-type: none;
+  line-height: 24px;
+  margin-bottom: 24px;
+}
+.about {
+  margin-top: 120px;
+  margin-left: 104px;
+}
+.property {
+  margin-top: 120px;
+}
+.support {
+  margin-top: 120px;
```

Comment:
Are u sure with copy-paste this property for every block? Why not to use it with wrapper?